ang/workflow/directives/workflow-list.directive.html

```diff
@@ -0,0 +1,50 @@
+<div id="bootstrap-theme">
+  <h1 crm-page-title>{{:: ts('Manage Workflows') }}</h1>
+
+  <div class="panel panel-default civicase-workflow-list">
+    <div class="panel-body">
+      <table class="table">
+        <thead>
+        <tr>
+          <th>{{:: ts('Title') }}</th>
+          <th>{{:: ts('Description') }}</th>
+          <th>{{:: ts('Enabled?') }}</th>
+          <th width="20">{{:: ts('') }}</th>
```

Comment:
what is this? what does `::` mean? why are we translating an empty string?

Reply:
Oops not needed. And `::` means its one way binding, and on digest cycles these values wont be refreshed.

Reply:
That's very good. Will check more into that. Thanks.